test/test_derived_parameters.py

```diff
@@ -0,0 +1,147 @@
+import numpy
```

Comment:
I agree its nice. Should it be apart of ``pycbc/tools/run_travis.sh`` though. I ran the test and it only took 0.26 seconds.